Capstone Project/PFObjects/Exercise.m

```diff
@@ -10,13 +10,13 @@
 @implementation Exercise
 
 @dynamic  name;
-@synthesize weight;
-@synthesize exerciseSet1;
-@synthesize exerciseSet2;
-@synthesize exerciseSet3;
-@synthesize exerciseSet4;
-@synthesize exerciseSet5;
-@synthesize workout;
+@dynamic weight;
```

Comment:
If this is dynamic then the setters and getters are not auto generated
You have to add them explicitly. Please dont proceed just because it is working somehow

-(void)setExerciseSet1:(NSInteger)value
{
  self.excerciseSet1 = value;
}

- (NSInteger)exerciseSet1
{
  return exerciseSet1;
}

Reply:
"No need if it works without that as documented"